manager/orchestrator/replicated/slot.go

```diff
@@ -29,7 +29,17 @@ func (is slotsByRunningState) Less(i, j int) bool {
 		}
 	}
 
-	return iRunning && !jRunning
+	if iRunning && !jRunning {
```

Comment:
Can I also request you to add a high level comment for the Less() function ? thx!